src/main/java/org/cbioportal/service/impl/AlterationCountServiceImpl.java

```diff
@@ -265,7 +265,13 @@ public List<CopyNumberCountByGene> getCnaGenes(StudyViewFilter studyViewFilter,
         var copyNumberCountByGenes = studyViewRepository.getCnaGenes(studyViewFilter, categorizedClinicalDataCountFilter);
         return populateAlterationCounts(copyNumberCountByGenes, studyViewFilter, categorizedClinicalDataCountFilter, AlterationType.COPY_NUMBER_ALTERATION);
     }
-    
+
+    @Override
+    public List<AlterationCountByGene> getStructuralVariantGenes(StudyViewFilter studyViewFilter, CategorizedClinicalDataCountFilter categorizedClinicalDataCountFilter) {
```

Comment:
someday i need to come to understand why we have to pass this categorizedClinicalDataCountFilter everywhere : )

Reply:
Actually we don't anymore. Get @onursumer  to remove it and push it down to the repository
https://github.com/cBioPortal/cbioportal/pull/10847/files#diff-4689c3a19361d9ee6a863aeceec6fbe3270f3e3df7764d7b6e089688829cded5R117-R140

Reply:
We needed it because there were 4 separate tables for clinical data